components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/connector/AdminForcedPasswordResetConfigImplTest.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.recovery.connector;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.governance.IdentityGovernanceException;
+import org.wso2.carbon.identity.recovery.IdentityRecoveryConstants;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.testng.Assert.assertEquals;
+
+/***
+ *This class does unit test coverage for AdminForcedPasswordResetConfigImpl class
+ */
+public class AdminForcedPasswordResetConfigImplTest {
+
+    private AdminForcedPasswordResetConfigImpl adminForcedPasswordResetConfigIml;
+
+    @BeforeTest
+    public void init() {
+        adminForcedPasswordResetConfigIml = new AdminForcedPasswordResetConfigImpl();
+    }
+
```

Comment:
Swap parameters in all assertEquals calls to make sure it follows the pattern `assertEquals(actualResut, expected)`